test/integration/targets/nxos_igmp_snooping/tests/common/sanity.yaml

```diff
@@ -39,6 +39,42 @@
           - "result.changed == false"
     when: (imagetag and (imagetag is version_compare('D1', 'ne')))
 
+  - block:
```

Comment:
Why do you specify a block for each test?  Can you do it all under the same block?

Reply:
pbkac. Fixed now.